### ISSUE TITLE
Page Layouts - User cannot use arrow key to change layout selection

### ISSUE THREAD

**Issue by bmattb** [GoogleForCreators/web-stories-wp#5911]
## Bug Description

When using keyboard nav the user is unable to use the arrow keys to move the selection of the currently selected page layout

## Expected Behaviour

User should be able to use the keyboard arrow keys to move the focus on the currently selected layout to another. 

## Steps to Reproduce

Within the keyboard layouts, try to move the selection to another with the keyboard arrow keys. You cannot.

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by BrittanyIRL** [comment]
Duplicates #5876
